components/infrastructure-services/backend/src/routes/kotlin/routes/product/DeleteProductInfrastructureService.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2023 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
```

Comment:
Is there a specific reason to set the copyright year not to 2025? (Also in other files.)

Reply:
It's a left-over from copying these files and just making some search & replace modifications. So, one could argue that most of the code existent before. But I'm also fine with updating the year as you prefer.